python/examples/workflows/multi_agents_simple.py

```diff
@@ -0,0 +1,56 @@
+import asyncio
+import traceback
+
+from beeai_framework.agents.types import AgentExecutionConfig
+from beeai_framework.backend.chat import ChatModel
+from beeai_framework.backend.message import UserMessage
+from beeai_framework.errors import FrameworkError
+from beeai_framework.memory import UnconstrainedMemory
+from beeai_framework.tools.search.wikipedia import WikipediaTool
+from beeai_framework.tools.weather.openmeteo import OpenMeteoTool
+from beeai_framework.workflows.agent import AgentFactoryInput, AgentWorkflow
+
+
+async def main() -> None:
+    llm = ChatModel.from_name("ollama:granite3.1-dense:8b")
+
+    try:
+        workflow = AgentWorkflow(name="Smart assistant")
+        workflow.add_agent(
+            agent=AgentFactoryInput(
+                name="Researcher",
+                instructions="You are a researcher assistant. Respond only if you can provide a useful answer.",
+                tools=[WikipediaTool()],
+                llm=llm,
+            )
+        )
+        workflow.add_agent(
+            agent=AgentFactoryInput(
+                name="WeatherForecaster",
+                instructions="You are a weather assistant. Respond only if you can provide a useful answer.",
+                tools=[OpenMeteoTool()],
+                llm=llm,
+                execution=AgentExecutionConfig(max_iterations=3, total_max_retries=10, max_retries_per_step=3),
+            )
+        )
+        workflow.add_agent(
+            agent=AgentFactoryInput(
+                name="Solver",
+                instructions="""Your task is to provide the most useful final answer based on the assistants'
+responses which all are relevant. Ignore those where assistant do not know.""",
+                llm=llm,
+            )
+        )
+
+        prompt = "What is the capital of France and what is the current weather there?"
+        memory = UnconstrainedMemory()
+        await memory.add(UserMessage(content=prompt))
+        response = await workflow.run(messages=memory.messages)
+        print(f"result: {response.state.final_answer}")
+
+    except FrameworkError:
+        traceback.print_exc()
+
+
+if __name__ == "__main__":
+    asyncio.run(main())
```

Comment:
Could you update this to match the other demos? Also I noticed that `workflows/emitter.py` also needs this updated if you could update that too (my mistake on a previous update)